CHANGELOG.org

```diff
@@ -4,6 +4,16 @@
 
 * Development
 
+Multi-path and wildcard handling contributed by Robert Weiner (@rswgnu):
+- Add support for grep and find over multiple directories and wildcarded paths.
+- Change =consult--directory-prompt=.  With a prefix arg sent as =dir= to one
+  of the consult search commands:
+    if it is a single C-u, prompt for a single directory to recursively search;
+    with any other prefix arg, prompt for a space-separated string of any
+    number of directories followed by file paths optionally with wildcards.
```

Comment:
I don't like the restriction that directories have to come before a list of wildcards. This restriction seems unnecessary and arbitrary. See also https://github.com/minad/consult/pull/749/files#r1116602923

Reply:
I think this was just related to removal of the the -r flag, which we have now done away with, so this restriction should no longer hold after the next push.